services/web/client/source/class/osparc/desktop/preferences/pages/SecurityPage.js

```diff
@@ -122,33 +121,49 @@ qx.Class.define("osparc.desktop.preferences.pages.SecurityPage", {
       return form;
     },
 
-    __createValidTokenForm: function(service, key, secret) {
-      let form = new qx.ui.form.Form();
+    __createValidTokenForm: function(token) {
+      const service = token["service"];
+
+      const height = 20;
+      const iconHeight = height-6;
+      const gr = new qx.ui.layout.Grid(10, 3);
+      gr.setColumnFlex(1, 1);
+      gr.setRowHeight(0, height);
+      gr.setRowHeight(1, height);
+      gr.setRowHeight(2, height);
+      const grid = new qx.ui.container.Composite(gr);
+
+      const nameLabel = new qx.ui.basic.Label(this.tr("Token name"));
+      grid.add(nameLabel, {
+        row: 0,
+        column: 0
+      });
 
-      let tokenService = new qx.ui.form.TextField().set({
-        value: service,
-        readOnly: true
+      const nameVal = new qx.ui.basic.Label(service);
+      grid.add(nameVal, {
+        row: 0,
+        column: 1
       });
-      form.add(tokenService, this.tr("Service API"));
 
-      let tokenKey = new qx.ui.form.TextField();
-      tokenKey.set({
-        value: key,
-        readOnly: true
+      /*
+      const showTokenIcon = "@FontAwesome5Solid/edit/"+iconHeight;
```

Comment:
Commented code can go?

Reply:
Ready for when the backend provides such a feature